src/reconstruct/helper.ts

```diff
@@ -1,20 +1,69 @@
-import { NodeId, ParentInformation, RNode, RType } from '../r-bridge';
-import { ReconstructionConfiguration } from './reconstruct';
+import { NodeId, ParentInformation, RNode, RType } from '../r-bridge'
+import { ReconstructionConfiguration } from './reconstruct'
 
 
 /*
 --helper function--
 */
 export type Selection = Set<NodeId>
+interface PrettyPrintLinePart {
+	part:   string
+	spaces: number
+}
 export interface PrettyPrintLine {
-	line: string
+	line:   string
 	indent: number
 }
-export function plain(text: string): PrettyPrintLine[] {
-	return [{ line: text, indent: 0 }]
+export interface WrappedPrettyPrintLine { //change name to be more fitting
+	linePart: PrettyPrintLinePart[]
```

Comment:
Couldn't we, from that perspective, simply replace `PrettyPrintLine` with this interface? why not always handle a line as an array of line parts (even if that array may have just one element).